style.css

```diff
@@ -174,3 +174,525 @@ header nav div .menu-bar {
+  flex-grow: 0;
+}
+
+.img-sec1 {
+  display: none;
+  height: 0;
+  width: 0;
+}
+
+.section-description1 {
+  display: none;
+  height: 0;
+  width: 0;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 0;
+  margin-top: 0;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
+}
+
+.experience {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 0;
+  height: 80px;
+  padding-bottom: 12px;
+}
+
+.section-title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 0;
+  padding-top: 12px;
+  width: 295px;
+  height: 36px;
+  white-space: nowrap;
+}
+
+.section-titlef {
+  display: none;
+  height: 0;
+  width: 0;
+}
+
+.list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  width: 236px;
+  height: 32px;
+  margin-top: 0;
+  white-space: nowrap;
+}
+
+.list1 {
+  list-style-type: none;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-right: 12px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.list2 {
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.list3 {
+  margin-left: 12px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  padding-top: 8px;
+  padding-bottom: 8px;
+}
+
+.section-description {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 12px;
+}
+
+.section-description0 {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 12px;
+  margin: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  padding-top: 12px;
+  list-style: none;
+}
+
+.li11 {
+  display: flex;
+  flex-direction: row;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 8px;
+}
+
+.li22 {
+  display: flex;
+  flex-direction: row;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 8px;
+}
+
+.li33 {
+  display: flex;
+  flex-direction: row;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 8px;
+}
+
```

Comment:
- [ ] Why are you repeating many CSS properties here for elements of the same kind? So why don't you give those element one class and use it to style them one which is acceptable as best practices but not what you have done here? Please remember the DRY(Dont respect yourself) principle. :smile:

Reply:
It's actually _**Don't repeat yourself**_. I believe it was a typo.